AsyncSwift/Observed/AppData.swift

```diff
@@ -9,8 +9,68 @@ import SwiftUI
 import UIKit
 
 final class AppData: ObservableObject {
-	@Published var currentTab: Tab = .event // Universal Link로 앱진입시 StampView 전환을 위한 변수
-    var scannedSeminarQR = true // Universal Link로 진입시 QR코드 스캔 여부
+    @Published var currentTab: Tab = .event // Universal Link로 앱진입시 StampView 전환을 위한 변수
```

Comment:
```suggestion
    /// Universal Link로 앱진입시 StampView 전환을 위한 변수
    @Published var currentTab: Tab = .event
```

`///`로 작성하면 바로 아래 property or method의 자동완성 설명에 뜬답니다 ✨

Reply:
commit 으로 수정했습니다!